src/picongpu/include/plugins/hdf5/WriteSpecies.hpp

```diff
@@ -154,9 +154,9 @@ struct WriteSpecies
                  mapper
                  );
             counterBuffer.deviceToHost();
-            log<picLog::INPUT_OUTPUT > ("HDF5:  ( end ) copy particle to host: %1%") % Hdf5FrameType::getName();
```

Comment:
Did you try to align the HDF5 with ADIOS? (+1 char)? :D

Am 10. Februar 2015 06:38:39 MEZ, schrieb f-schmitt-zih <notifications@github.com>:
>> @@ -154,9 +154,9 @@ struct WriteSpecies
>>                   mapper
>>                   );
>>              counterBuffer.deviceToHost();
>> -            log<picLog::INPUT_OUTPUT > ("HDF5:  ( end ) copy
>particle to host: %1%") % Hdf5FrameType::getName();
>
>all those whitespaces were actually intended to create a little more
>hierarchy in the output
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ComputationalRadiationPhysics/picongpu/pull/679/files#r24390224